### ISSUE TITLE
Not able to genrate report for publisher-ads-audits using : https://pub-ads-audits.appspot.com/

### ISSUE THREAD

**Issue by virendra-rastogi** [googleads/publisher-ads-lighthouse-plugin#347]
Include the following information

- I am not able to generate  publisher-ads-audits report for any of the url from times of india domain
- We should get audit report
- Steps to reproduce
  - [Environment (CLI or web service)] We are using https://developers.google.com/publisher-ads-audits/
  - URL : Testing domain : https://timesofindia.indiatimes.com
![Screenshot from 2022-05-26 14-36-29](https://user-images.githubusercontent.com/101269278/170457695-81616794-c2e6-40dd-b70d-d077ede9ceea.png)

**Comment by mcmd** [comment]
I was able to reproduce with `https://khaleejtimes.com/`. It worked with `https://elpais.com/m though.

**Comment by virendra-rastogi** [comment]
@mcmd Yes its working with other domains of my company as well not with the domain  : https://timesofindia.indiatimes.com what could be the possible reason
